modules/dgi_migrate_foxml_standard_mods/migrations/dgis_nodes.yml

```
@@ -2592,6 +2619,76 @@ process:
         - '@_use_license'
         - '@_unspecified_use_license'
     - plugin: flatten
+  _rights_statement_query:
+    - <<: *base_mods_node
+      query: 'mods:accessCondition[@type="rights statement"]'
+    - plugin: callback
+      callable: iterator_to_array
+  _rights_statement:
+    - plugin: get
+      source: '@_rights_statement_query'
+    - plugin: skip_on_empty
+      method: process
+    - plugin: multiple_values
+    - plugin: dgi_migrate.sub_process
+      process_values: true
+      values:
+        _vid:
+          - plugin: default_value
+            default_value: rights_statements
+        <<: *generic_term_after
+    - <<: *generic_term_extract
+    - plugin: single_value
+    - plugin: callback
+      callable: array_filter
+    - plugin: null_coalesce
+  _unspecified_rights_statement:
+    - plugin: get
+      source: '@_rights_statement_query'
+    - plugin: skip_on_empty
+      method: process
+    - plugin: get
+      source: '@_unspecified_term'
+    - plugin: gate
+      use_as_key: '@_rights_statement'
+      valid_keys:
+        - null
+      key_direction: unlock
+    - plugin: dgi_migrate.sub_process
+      process_values: true
+      values:
+        _vid:
+          - plugin: default_value
+            default_value: rights_statements
+        _auth_value_uri:
+          - plugin: default_value
+            default_value: ''
+        _auth_source:
+          - plugin: default_value
+            default_value: ''
+        _value:
+          - plugin: get
+            source: parent_value
+        actual:
+          - plugin: get
+            source:
+              - '@_auth_source'
+              - '@_auth_value_uri'
+              - '@_value'
+              # XXX: Needs to be provided wherever this is used, corresponding
+              # to the vocab in which to do the things.
+              - '@_vid'
+          - plugin: flatten
+          - plugin: migration_lookup
+            migration: dgis_stub_terms_generic
+            stub_id: dgis_stub_terms_generic
```

Half-started thinking this could do the:

```suggestion
        <<: *generic_term_after
```

thing, but not so much because we don't have the DOM node from which to query the misc values (such as `@valueURI` and `@authority` and the text content of the given node), so yeah, is fine.